machine.go

```diff
@@ -323,8 +324,13 @@ func (m *Machine) writerKernelModules(w *writerhelper.WriterHelper) error {
 		if mergedUsrSystem() {
 			usrpath = "/usr/lib/modules"
 		}
-		if err := w.CopyFile(path.Join(usrpath, kernelRelease, v)); err != nil {
-			return err
+		log.Printf("Getting module from host %s", v)
+		if (!strings.HasPrefix(v, "kernel/drivers/virtio")) {
+			if err := w.CopyFile(path.Join(usrpath, kernelRelease, v)); err != nil {
+				return err
+			}
+		} else {
+			log.Printf("Ignoring %s", v)
```

Comment:
Would be better to print a message about a module not being found and assuming it's build in (even better if we could determine that for sure, but tha'ts not always possible)